packages/app/src/debug/DebugPendingRunSplash.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div
+    data-cy="debug-pending-splash"
+    class="space-around flex"
+  >
+    <div class="flex flex-col w-full items-center">
+      <IconTechnologyDashboardRunning class="mb-16px" />
+      <span
```

Comment:
Nothing to change here I don't think, but I'm wondering what people's thoughts are on using test IDs for things like this. I usually try to stay away from using `data-cy` if I can, since users don't see test IDs on screen.

Is there a reason why we can't just grab the title here with `cy.contains` or something like that, since that's what the user would actually see on the page? I understand this makes things a little more brittle (for instance if we decide to update the text then the test will fail). Just asking because I'm trying to refine my own testing strategy.